tests/resources/complete_config.yaml

```diff
@@ -1,5 +1,8 @@
+### url of schema
+schema: https://raw.githubusercontent.com/snehasi/simple_grid_site_repo/master/simple_base_schema.yaml
```

Comment:
We can update this to the main site_repo now ;)

Reply:
Cool, done!